client/src/plugins/version-info/ReleaseInfo.js

```diff
@@ -44,20 +44,20 @@ export function ReleaseInfo(props) {
     <div className={ css.ReleaseInfo }>
       <ul className="dashed">
         <li>
-          <h4>Model all elements for Camunda 8</h4>
-          The BPMN editor now supports drawing all BPMN symbols while it also indicates which elements are not supported in the engine.
+          <h4>New form fields and more in the Form editor</h4>
+          The form editor now supports text area, datetime and image form fields. Add conditions to form fields to control visibility.
```

Comment:
Let's use the oxford comma in accordance with the [Camunda Writting Style guide](https://confluence.camunda.com/display/HAN/Camunda+Writing+Style+Guide#CamundaWritingStyleGuide-Punctuation) 
```suggestion
          The form editor now supports text area, datetime, and image form fields. Add conditions to form fields to control visibility.
```

Reply:
TIL about the Oxford comma. 🐂